src/virtual-wan/azext_vwan/_params.py

```diff
@@ -121,7 +121,9 @@ def load_arguments(self, _):
         c.argument('shared_key', help='Shared key.')
 
     with self.argument_context('network vpn-gateway connection list') as c:
+        # List commands cannot use --ids flag
         c.argument('resource_name', vpn_gateway_name_type, id_part=None)
+        c.argument('gateway_name', id_part=None)
```

Comment:
Is it a breaking change, as the 'gateway_name' parameter required?

Reply:
As the comments said, if I don't set `id_part=None`. The argument lists would have `--ids` for list command.

After using the origin SDK method as the implementation method and setting id_part of `gateway_name` to None and , it behaves like the same as before

![image](https://user-images.githubusercontent.com/14357159/100567589-cd775380-3303-11eb-95cb-f89564c9bf2d.png)

![image](https://user-images.githubusercontent.com/14357159/100567605-da944280-3303-11eb-96e8-af16b7f6f0e7.png)